### PR TITLE
Update pro page 

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Update /pro 

## QA
- Go to https://ubuntu-com-13104.demos.haus/pro
- Compare page against [Figma](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?type=design&node-id=4091-20096&mode=design&t=KcoJMipE5vifQ0h8-0) and [copy doc](https://docs.google.com/document/d/1XidZq045nJDM5vzfB5KJCpH3AqOERvPOeayuqUKfNso/edit#heading=h.i6h9ir8nv0px) -> Page on Demo might be different from design on Figma as PdM requested some changes.
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-4517

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by minkyngkm** [comment]
@leiaru-c or @lyubomir-popov For this reference section, I used the [tab](https://vanillaframework.io/docs/patterns/tabs) component from vanilla and tab width is different depending on logo width. Could you please help me get logos for this section? 🙂 Please see this [copy doc](https://docs.google.com/document/d/1XidZq045nJDM5vzfB5KJCpH3AqOERvPOeayuqUKfNso/edit#heading=h.i6h9ir8nv0px) for your reference.

image

**Comment by unco** [comment]
image

Heading breaks oddly on mobile

**Comment by unco** [comment]
image

Spacing is too close to the group above the title here on mobile

**Comment by unco** [comment]
image

This might be one for @juanruitina  but should the term 'Ubuntu Pro' be 'Ubuntu&nbsp;Pro' so that we don't get orphans like this?

**Comment by unco** [comment]
Oops Ubuntu - non-breaking space - Pro

**Comment by juanruitina** [comment]
@unco I agree,`&nbsp;` it :)

**Comment by minkyngkm** [comment]
Thanks @unco and @juanruitina  for your comments! 🙏
1. For the odd heading breaks -> made `<br />` hidden on mobile. 
2. Spacing too close -> added more space below the group of logos 
3. "Ubuntu Pro" word break -> `&nbsp` added!

Hopefully it looks better now! Thanks a lot again! 👍

File: templates/pro/index.html [hunk 1]
```diff
@@ -1,865 +1,749 @@
...
-      ) | safe
-      }}
+  </div>
+  <hr class="is-fixed-width u-no-margin--bottom">
+  <div class="row--50-50 u-sv3">
+    <div class="col">
+      <h3 class="p-heading--2">Live kernel updates</h3>
+    </div>
+    <div class="col">
+        <p>Minimize downtime and unplanned reboots with patches for critical and high-severity kernel vulnerabilities.</p>
```
Comment (albertkol) [inline templates/pro/index.html:475]: Minimise*

**Comment by unco** [comment]
I think that Ubunto Pro with non breaking space should sort of be the
default because it’s hard to imagine a scenario where you wouldn’t want
those to be unbroken

**Review by albertkol — APPROVED** [review]
It's pretty :star:

**Comment by minkyngkm** [comment]
@unco Thanks Hayden. Yep that's a good point 👍 I've updated all Ubuntu Pro with non breaking space on this pro page. We  will try to update them for all the pages on ubuntu.com in the future.

**Comment by lyubomir-popov** [comment]
This is not as in the design:

image

Please let us know when someone requests such a big departure from the agreed design, no point building something that will need to be redesigned.

**Comment by lyubomir-popov** [comment]
Awesome work @minkyngkm thank you!

**Pull request merged**